src/main/java/org/opensrp/service/ClientService.java

```diff
@@ -248,8 +249,9 @@ public Long countAll(long serverVersion) {
 		return allClients.notInOpenMRSByServerVersion(serverVersion, calendar);
 	}
 
-	@PreAuthorize("hasRole('CLIENT_VIEW')")
-	@PostFilter("hasPermission(filterObject, 'CLIENT_VIEW')")
+	/*
+	No ACL to allow for global searching
```

Comment:
can we add more details on why this is being excluded, also the exclusion should be via a new method